docs/usage/updating-rebasing.md

```diff
@@ -1,53 +1,62 @@
 ---
-title: Updating and Rebasing Branches
-description: How Renovate Updates and Rebases Branches
+title: Updating and Rebasing branches
+description: How Renovate Updates and Rebases branches
 ---
 
-# Updating and Rebasing Branches
+# Updating and Rebasing branches
 
-There are many cases where Renovate will need to update a branch/PR after its initial creation, and this document will attempt to describe them.
+There are many situations in which Renovate must update/rebase a branch.
 
-Note: Renovate doesn't technically do "rebasing" in the Git sense.
-Instead, it manually recreates the same commit based off of the latest commit in base branch.
+Here is a list of the most common cases where Renovate must update/rebase the branch:
 
-## If you have made edits
+- When a pull request has conflicts due to changes on the base branches
+- When you have enabled "Require branches to be up to date before merging" on GitHub
+- When you have manually told Renovate to rebase when behind the base branch with `"rebaseWhen": "behind-base-branch"`
+- When a newer version of the dependency is released
+- When you request a manual rebase from the Renovate bot
+
+Renovate uses its own version of "rebasing", which is _not the same_ as doing a `git rebase` with Git.
+Instead of doing a `git rebase`, Renovate manually recreates the PR commit based off of the latest commit in the base branch.
```

Comment:
```suggestion
```

Scrap this section see https://github.com/renovatebot/renovate/pull/8029#issuecomment-745314007 for my reasoning.